velox/type/Timestamp.h

```diff
@@ -169,8 +176,10 @@ struct Timestamp {
 
   /// Due to the limit of std::chrono, throws if timestamp is outside of
   /// [-32767-01-01, 32767-12-31] range.
+  /// If allowOverflow is true, integer overflow is allowed in converting
+  /// timestmap to milliseconds.
```

Comment:
What does it mean that overflow is allowed? What happens on overflow? Are we wrapping around?

Reply:
Yes, wraparound happens. 
The result for fromUnixTime(INT64_MAX, "yyyy-MM-dd HH:mm:ss") is "1969-12-31 23:59:59", since -1000 milliseconds is produced when converting INT64_MAX seconds to milliseconds.